simtools/testing/assertions.py

```diff
@@ -41,3 +43,109 @@ def assert_file_type(file_type, file_name):
     # no dedicated tests for other file types, checking suffix only
     _logger.info(f"File type test is checking suffix only for {file_name} (suffix: {file_type}))")
     return Path(file_name).suffix[1:] == file_type
+
+
+def assert_n_showers_and_energy_range(file):
+    """
+    Assert #showers and energy range.
```

Comment:
What does `#` mean?

Reply:
And maybe say that you look into a log file of corsika or simtelarray here?

Reply:
`#` means number and I think I wrote it because I didn't want to repeat myself with the word "number". I changed it back to number.
This is one of those cases in which I had to reformulate because ruff wouldn't let me have one long sentence which took more than one line...


I do not look into the log files of CORSIKA and sim_telarray here but instead read the sim_telarray output file itself and extract the headers and count the `#` of events and energies produced.

Reply:
Still don’t understand why you can’t use `number`. The first line is not that long.

Reply:
I changed it already to be `Assert the number of showers and the energy range.`

My complaint was about the original version you didn't see which was 
```
Assert the number of showers is consistent with the required one (up to 1% tolerance) 
and that the energies simulated are within the configured ones.
```

This is too long to be one line and ruff doesn't allow a summary line with a line break in it, so I had to break it into two separate statements with some repetition. I find it a bit annoying, so I complained.